CreditCardFraud/README.md

```diff
@@ -0,0 +1,34 @@
+# Credit Card Fraud Data
+
+This directory contains a [credit card purchase dataset](https://www.kaggle.com/mlg-ulb/creditcardfraud).  The data set contains anonymized parameters for 48 hours worth of purchases by European cardholders.  There are 284,807 purchases in total, and 492 of them are fraudulent.
+
+## Fields in `creditcard.csv`
+
+The `creditcard.csv` file contains 31 columns of data.  These columns are:
+
+- **Time** - The integer number of seconds after the first purchase in the file.
+- **V1 - V28** - The 28 anonymized PCA transformed purchase parameters.
+- **Amount** - The purchase amount in Euros.
+- **Class** - A flag indicating a valid (0) or fraudulent (1) purchase
+
+# Source and license
+
+This data is taken directly from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.  Credit for the dataset goes to:
```

Comment:
```suggestion
This data is taken directly from [Kaggle](https://kaggle.com).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.  

Credit for the dataset goes to:
```

Reply:
This looks more like a Works Cited List. How about, "The following sources contributed to this dataset:"